settings.gradle

```diff
@@ -1 +1,4 @@
-include ':gnd'
+include ':gnd', ':android-gmaps-addons'
+
+project(':android-gmaps-addons').projectDir = new File('third_party/android-gmaps-addons')
```

Comment:
Can we nest this isn't another directory w/the repo/user name so that it's clearer that it's a copy of that repo?
i.e. `third_party/cocoahero/android-gmaps-addons`

Reply:
Sure thing!